x/datapool/types/expected_keepers.go

```diff
@@ -4,11 +4,19 @@ import (
 	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/cosmos/cosmos-sdk/x/bank/exported"
 )
 
 type BankKeeper interface {
 	GetBalance(ctx sdk.Context, addr sdk.AccAddress, denom string) sdk.Coin
 	SendCoins(ctx sdk.Context, fromAddr sdk.AccAddress, toAddr sdk.AccAddress, amt sdk.Coins) error
+	SendCoinsFromModuleToAccount(ctx sdk.Context, senderModule string, recipientAddr sdk.AccAddress, amt sdk.Coins) error
+
+	MintCoins(ctx sdk.Context, moduleName string, amt sdk.Coins) error
+	BurnCoins(ctx sdk.Context, name string, amt sdk.Coins) error
```

Comment:
It doesn't need this feature right now. However, since the minted shareToken may be deleted later, so I added it first.
Additionally, you need to add the authtypes.Burner permission.

Reply:
When do we need `BurnCoins()`? maybe revoke pool? Do you have any idea about this?

Reply:
When do we need `BurnCoins()`? maybe revoke pool? Do you have any idea about this?

Reply:
I haven't really thought about it in detail yet.
Maybe refund?